hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/ConfirmationDialog.java

```diff
@@ -8,6 +8,7 @@
  */
 package org.eclipse.hawkbit.ui.common;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Please use Guava's `Strings` instead of commons-lang. `Strings.nullOrEmpty(final String s)`